src/crs_linter/linter.py

```diff
@@ -797,6 +817,24 @@ def check_crs_tag(self):
                             "message": f"rule does not have tag with value 'OWASP_CRS'; rule id: {ruleid}",
                         }
                     )
+                # see the exclusion list of files which does not require the filename tag
+                if filenametag not in ["OWASP_CRS/crs-setup.example", \
```

Comment:
This should be an argument. We do the same thing basically with `APPROVED_TAGS`.

Reply:
Currently we use `APPROVED_TAGS` from CRS's repository (but the linter has it's own). Where do we want to store this? Here, under the linter, or under CRS?

Reply:
In the CRS repo. Configuration should live where it is versioned together with the stuff it is used for. See for example the `toolchain.yaml` for the toolchain in the CRS repo.

Reply:
Another idea: now `APPROVED_TAGS` contains all possible tags, right (under the `coreruleset` repository)?

What if the script tried to figure out the necessary tag at the rule from the filename? Cons: could it be there any possible collision?

Reply:
I would say "yes", but in the future (hopefully not too far away) we will start generating rules and the filenames may loose their relevance. At that point the script would no longer work. So for now, I'd keep working with the file.

Reply:
Sorry for the long delay, I added that modification, so from now the linter allows an (optional) argument (a filename) which contains the list of files where we don't expect filename tags.

Now we have to solve that add this file from linter's repository as a cli argument during the regression tests.

Reply:
Okay, it's done.